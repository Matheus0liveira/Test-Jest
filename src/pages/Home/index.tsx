export default function Home({ title = 'Matheus' }) {
  return (
    <header>
      <h1>{title}</h1>
    </header>
  );
}
