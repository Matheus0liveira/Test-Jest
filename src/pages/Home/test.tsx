import { render, screen } from '@testing-library/react';

import Home from '.';

describe('<Main/>', () => {
  it('Render the Head', () => {
    const { container } = render(<Home />);

    expect(
      screen.getByRole('heading', { name: /Matheus/i })
    ).toBeInTheDocument();

    expect(container.firstChild).toMatchSnapshot();
  });
});
