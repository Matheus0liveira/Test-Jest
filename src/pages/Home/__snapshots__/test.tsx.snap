// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main/> Render the Head 1`] = `
<header>
  <h1>
    Matheus
  </h1>
</header>
`;
